### PR TITLE
A0M-88: Appropriate message in search bar when searching for updates and through all modules

### PR DESCRIPTION
**Pull request by Annettesunday**
## JIRA TICKET NAME:

[AOM-88 Appropriate message in search bar when searching for updates and through all modules](https://issues.openmrs.org/browse/AOM-88)

## SUMMARY:
A user should be able to view appropriate messages in search bar when searching for updates and through all modules

### CODE REVIEW

**Comment by Farhanisto** [comment]
Looks good

**Comment by mkiterian** [comment]
Looks good 👍

**Comment by kodero** [comment]
Good.

**Comment by patlub** [comment]
Looks fine

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -671,7 +671,9 @@ export default class ManageApps extends React.Component {
                     id="search-input"
                     ref={value => this.input = value}
                     onKeyDown={this.initiateSearch}
-                    placeholder="Need new add-ons? Type & press ENTER or clear field to display installed add-ons" />
+                    placeholder={!updatesAvailable ? 
+                      "Need new add-ons? Type & press ENTER or clear field to display installed add-ons" :
```
Comment (LWanjiru) [inline app/js/components/manageApps/ManageApps.jsx]: Just curious... when you say `type & press Enter`, is there a specification of what they should be typing? A name, description?

**Pull request merged**